### PR TITLE
Updated profiles to new bioschemas types and fixed some typos

### PR DESCRIPTION
**Pull request by floWetzels**
I updated the profile according to the new bioschemas type and specification adaptations. I also fixed some typos/bugs and added bioschemas/schema.org tags before type names. Please have a look, @HLWeil @stuzart

### CODE REVIEW

File: profile/isa_ro_crate.md [hunk 1]
```diff
@@ -195,17 +197,17 @@ Describes and points to a Data file, and maps to the [ISA-JSON Data](https://isa
 |@type |MUST|Text|must be 'File' or 'MediaObject'|
```
Comment (HLWeil) [inline profile/isa_ro_crate.md:197]: Add link to types here like with the other profiles.

File: profile/isa_ro_crate.md [hunk 2]
```diff
@@ -69,122 +73,120 @@ Protocol --reagent---> ont
...
+|variableMeasured|COULD|Text or [schema.org/PropertyValue](https://schema.org/PropertyValue)|The target variable being measured E.g protein concentration|
 |dateCreated|SHOULD|DateTime|When the Assay was created|
 |dateModified|SHOULD|DateTime|When the Assay was last modified|
-|citation|COULD|ScholarlyArticle|A publication corresponding to this assay.|
-|comment|COULD|Comment|Comment|
+|citation|COULD|[schema.org/ScholarlyArticle](https://schema.org/ScholarlyArticle)|A publication corresponding to this assay.|
+|comment|COULD|[schema.org/Comment](https://schema.org/Comment)|Comment|
 
 
 ### LabProcess (_new type_)
```
Comment (HLWeil) [inline profile/isa_ro_crate.md:142]: The `new type` is not needed anymore I think.

File: profile/isa_ro_crate.md [hunk 3]
```diff
@@ -69,122 +73,120 @@ Protocol --reagent---> ont
...
 
-Has the new suggested Bioschemas LabProcess type and maps to the [ISA-JSON Process](https://isa-specs.readthedocs.io/en/latest/isajson.html#process-schema-json)
+Has the new suggested Bioschemas [bioschemas.org/LabProcess](https://bioschemas.org/LabProcess) type and maps to the [ISA-JSON Process](https://isa-specs.readthedocs.io/en/latest/isajson.html#process-schema-json)
```
Comment (HLWeil) [inline profile/isa_ro_crate.md:144]: Maybe change to `Has the Bioschemas DRAFT type and...`

Comment (HLWeil) [reply profile/isa_ro_crate.md:144]: Not necessary, but the LabProcess is on the same level like LabProtocol and such now.

**Review by HLWeil — CHANGES_REQUESTED** [review]
Links look really good like this!

**Review by stuzart — COMMENTED** [review]
sorry, I only just spotted this but looks good

**Pull request merged**